### DISCUSSION TITLE
On the fly similar to flare++

### DISCUSSION THREAD

**Discussion by deleep225** [mir-group/nequip#193] [Ideas]
Is there a way to do the on the fly development of force field when the energies and forces are going far from realistic value similar to the below example?
https://github.com/mir-group/flare_pp.

**Comment by simonbatzner** [comment]
Hi @deleep225, currently no such functionality exists. What you could of course do is run FLARE, generate a training set and then train NequIP on that. This is likely to give you a really good+diverse training set.

**Reply by deleep225** [reply]
Thank you @simonbatzner I will try it.
